backend/src/main/java/com/hyooun/wilson/common/PersistenceAdapter.java

```diff
@@ -0,0 +1,17 @@
+package com.hyooun.wilson.common;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.stereotype.Component;
+
+import java.lang.annotation.*;
+
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Component
+public @interface PersistenceAdapter {
+
+    @AliasFor(annotation = Component.class)
+    String value() default "";
+
```

Comment:
PersistenceAdapter 어노테이션에 @AliasFor를 추가하여 Component 어노테이션의 value 속성을 설정하세요.

```suggestion
@AliasFor(annotation = Component.class)
String value() default "";
```